docs/temboard-upgrade-5-6.0.md

```diff
@@ -9,8 +14,8 @@ sudo systemctl stop temboard
 Update temboard package with your [preferred installation
 method](installation.md#installation).
 
-
-Stamp `repository` database schema to latest version:
+temBoard 5.X database is compatible with temBoard 6.0. However, you must stamp
+`repository` database for temBoard to consider it updated:
```

Comment:
What do you think about the following:
```
Even though the database structure hasn't changed, it is now required to version the schema. This will make the future updates a lot easier.

The `repository` database schema must be stamped to the latest version otherwise temBoard will not run:
```